library/network/src/lib/y2firewall/firewalld/interface.rb

```diff
@@ -96,6 +96,15 @@ def zone
         fw.zones.find { |z| z.interfaces.include?(name) }
       end
 
+      # Assign the interface to the given zone
+      #
+      # @param zone_name [String] the name of the zone to be assigned to
+      def zone=(zone_name)
+        fw.zones.map { |z| z.remove_interface(name) if z.interfaces.include?(name) }
```

Comment:
Where is `name` defined?

Reply:
https://github.com/yast/yast-yast2/blob/f318a9239b0ec015610f75c5c05f85123650222e/library/network/src/lib/y2firewall/firewalld/interface.rb#L78

Reply:
Oops! Apologies :) I must to wake up as soon as possible. 

Sorry for the noise.